modules/gathering-clusterversion-history-cli.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * updating/troubleshooting_updates/gathering-data-cluster-update.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gathering-clusterversion-history-cli_{context}"]
+= Gathering ClusterVersion history using the {oc-first}
+
+To view the ClusterVersion history using the {oc-first}:
+
+.Prerequisites
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have installed the {oc-first}.
+
+.Procedure
+
+. View cluster update history:
```

Comment:
```suggestion
. View the cluster update history by entering the following command:
```

Reply:
See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#procedure-step-that-introduces-a-command:

"If a step in a procedure is to run a command, make sure that the step text includes an explicit instruction to "run" or "enter" the command."